spaces/S000174/README.md

```diff
@@ -6,5 +6,8 @@ refs:
     name: "A perfectly normal locally metrizable non-paracompact space"
 ---
 
+Let $X$ be a metrizable topological space admitting a basis of cardiality $\aleph_1$. Let $(X_\xi)_{\xi < \aleph_1}$ be an increasing transfinite sequence of closed, {P27} subsets of $X$, s.t.,
```

Comment:
For simplicity, could this starting space be a discrete space of cardinality $\aleph_1$? (Such a space isn't listed in pi-Base unless you assume the continuum hypothesis, I'm realizing.)

Generally we reserve $X$ for the space being defined; rather than re-topologizing $X$, perhaps we can call the starting space $Y$ (or $\omega_1$?) then define a topology for $X$ without "overwriting" the previous $X$?

Reply:
I think I'm missing something here; if $X$ is discrete $\omega_1$, then $X_\xi$ could be $\xi\subseteq\omega_1$, but then I think we recover the original discrete topology through this process.

Reply:
@StevenClontz I'll change $X$ to $Y$, but yeah, that is weird. I thought I was copying the paper's construction nearly verbatim, but perhaps I missed something. Let me check again.

Reply:
Oh, I realized what went wrong. The general construction is described as I stated. And the resulting $X$ will be $T_6$, locally metrizable, and collectionwise normal, regardless of what $X$ we started with is. But the final property, non-paracompact, that's asserted in the paper, only applies to $X$ with some specific property. The paper didn't mention those additional requirements until the proposition asserting non-paracompactness, so I just missed that. I'll add that in.

Reply:
Basically, one has to ensure that each $X_\xi$ actually adds something, i.e., $X_\xi \setminus \left(\bigcup_{\alpha<\xi} X_\alpha\right) \neq \varnothing$, even for limit ordinals $\xi$. And this is not true for discrete space $\omega_1$. The example given in the paper is instead $\omega_1^\omega$, where $\omega_1$ is equipped with the discrete topology. I'll just add in this specific example for concreteness.